.github/workflows/ci.yml

```diff
@@ -37,9 +37,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        activerecord: [ '5.2', '6.0', '6.1', '7.0', '7.1' ]
-        pg: [ '1.1', '1.2' ]  # FU: Build against common pg 1.x minor versions
-        ruby: [ '2.7' ]  # FU: Build against common ruby 3.x minor versions
+        activerecord: [ '6.1', '7.0', '7.1' ]
+        pg: [ '1.5' ]
```

Comment:
Do we still need to specify this, or can we remove it and the corresponding entries from the `Appraisals` in favor of using the latest available version on Rubygems?

Reply:
We don't strictly need it. I left it in for convenience but I'll remove it as YAGNI.